streaming_providers/routes.py

```diff
@@ -131,6 +134,56 @@ async def streaming_provider_endpoint(
 
     await release_redis_lock(lock)
 
+    # If proxying debrid is enabled, proxy the link
```

Comment:
@ragmehos lets set up a separate endpoint for proxy users.  `/streaming_provider/{secret_str}/proxy_stream`   and we can directly call streaming_provider_endpoint function and use that link inside the proxy_stream.  

So that this endpoint can be distinguished from any subscriptional block.

Reply:
Done, can you take a look?